libraries/matware/jupgrade/schemas/1.5/contents.php

```diff
@@ -179,8 +179,8 @@ public function dataHook($rows = null)
 			// @@ Duplicate entry for key 'idx_client_id_parent_id_alias_language'
 			if ($content->load(array('alias' => $row['alias'], 'catid' => $row['catid'])))
 			{
-				// Getting the duplicated alias
-				$alias = $this->getAlias('#__content', $row['alias']);
+				$content->reset();
+				$content->id = 0;
 				// Set the modified alias
 				$row['alias'] .= "-".rand(0, 99999999);
```

Comment:
@klas Yes i see the changes but some people like to preserve 'id' to prevent SEF urls breaks, if you remove the id this could broke that. I'm not sure but need to test it